src/components/_button.scss

```diff
@@ -180,6 +213,21 @@
   font-size: $fz-l;
 }
 
+@mixin sui-button-icon--small {
```

Comment:
Why mixins for doing this? It's simpler to create vars like: `$size-icon-small, $size-icon-medium...`